.ci/azure-pipelines.yml

```diff
@@ -0,0 +1,183 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+variables:
+  - name: TestProjects
+    value: 'Jellyfin.Server.Tests/Jellyfin.Server.Tests.csproj'
+  - name: RestoreBuildProjects
+    value: 'Jellyfin.Server/Jellyfin.Server.csproj'
+
+pr:
+  autoCancel: true
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+
+jobs:
+  - job: main_build
+    displayName: Main Build
+    pool:
+      vmImage: ubuntu-16.04
+    strategy:
+      matrix:
+        release:
+          BuildConfiguration: Release
+        debug:
+          BuildConfiguration: Debug
+      maxParallel: 2
+    steps:
+    - checkout: self
+      clean: false
```

Comment:
It should be less important here since we aren't using local workers or doing terrible things to git in this pipeline, but in my experience with a smaller project the time saved by not having clean enabled is not worth the chances of the worker accidentally getting in a bad state.